### PR TITLE
Improved skip nav

### PR DESCRIPTION
**Pull request by alany411**
- Added styling to skip nav to fix display issue when in focus

Picture of issue:
image

### CODE REVIEW

File: components/Container.js [hunk 1]
```diff
@@ -45,14 +45,14 @@ export default function Container(props) {
...
-        <a href="#skip" className="sr-only focus:not-sr-only">
+      <nav className="flex items-center justify-between w-full max-w-4xl p-8 mx-auto my-0 text-gray-900 bg-white sticky-nav md:my-8 dark:bg-black bg-opacity-60 dark:text-gray-100">
+        <a href="#skip" className="skip-nav">
           Skip to content
         </a>
         <button
           aria-label="Toggle Dark Mode"
           type="button"
-          className="bg-gray-200 dark:bg-gray-800 rounded p-3 h-10 w-10"
+          className="w-10 h-10 p-3 bg-gray-200 rounded dark:bg-gray-800"
```
Comment (leerob) [inline components/Container.js:55]: Why did these change?

File: styles/global.css [hunk 2]
```diff
@@ -37,6 +37,10 @@ html {
   transition: background-color 0.1 ease-in-out;
 }
 
+.skip-nav {
+  @apply absolute px-4 py-2 transition-transform duration-200 transform -translate-y-12 focus:translate-y-3 -top-8;
```
Comment (leerob) [inline styles/global.css:41]: Doesn't it still need the sr classes?

File: components/Container.js (hunk 1 above)
Comment (alany411) [reply components/Container.js:55]: I use the [headwind](https://github.com/ryanhhhh/headwind) extension and it sorted the classes.

File: styles/global.css (hunk 2 above)
Comment (alany411) [reply styles/global.css:41]: The sr classes are mainly used just to hide and show an element visually without fully hiding them from a screen reader. [https://tailwindcss.com/docs/screen-readers](https://tailwindcss.com/docs/screen-readers).

**Review by leerob — APPROVED** [review]
Thank you!

**Pull request merged**